aws-auth-cognito/src/test/resources/feature-test/states/SignedIn_SessionEstablished.json

```diff
@@ -11,9 +11,9 @@
                 "authType": "USER_SRP_AUTH"
             },
             "cognitoUserPoolTokens": {
-                "idToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyfQ.WK8_fwfvSU7wdDl5sovnRApzUp6FHNB5ljS8KAO0QeA",
-                "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyfQ.WK8_fwfvSU7wdDl5sovnRApzUp6FHNB5ljS8KAO0QeA",
-                "refreshToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyfQ.WK8_fwfvSU7wdDl5sovnRApzUp6FHNB5ljS8KAO0QeA",
```

Comment:
are these changes required?

Reply:
@gpanshu Yes to all of these questions. These files should have been modified with the AuthGenerator, but it looks like previous PR some of these were manual. So when I made my token changes, I made sure all the updates were applied according to the generator. We should never edit these json files manually.